aztk/spark/client.py

```diff
@@ -21,10 +21,8 @@ class Client(BaseClient):
     Args:
         secrets_config(aztk.spark.models.models.SecretsConfiguration): Configuration with all the needed credentials
     """
-    def __init__(self, secrets_config):
-        super().__init__(secrets_config)
 
-    def create_cluster(self, cluster_conf: models.ClusterConfiguration, wait: bool = False):
+    def create_cluster(self, configuration: models.ClusterConfiguration, wait: bool = False):
```

Comment:
I think we should not do this since this is a breaking change. In the sdk rewrite, we will make sure the parameter names are better. But for now, we should preserve backwards compatibility and then deprecate these methods.